docs/tutorials/google/compare_to_calibration.ipynb

```diff
@@ -0,0 +1,760 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    def create_loschmidt_echo_circuit(
</pre><p><code>create_loschmidt_echo_circuit</code>   function here is implemented differently from the linked reference tutorial https://quantumai.google/cirq/tutorials/google/echoes#creating_the_circuits.</p><p> </p><p>Maybe a description about the implementation would be helpful here as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>